specs/deposits.md

```diff
@@ -193,6 +193,9 @@ A predeployed contract on L2 at address `0x4200000000000000000000000000000000000
 certain block variables from the corresponding L1 block in storage, so that they may be accessed
 during the execution of the subsequent deposited transactions.
 
+A `seqnr` attribute is stored along with the L1 attributes to track the L2 block distance
+to the L2 block height that the L1 attributes were first introduced.
```

Comment:
I think a clearer description would be

> A `seq_number` attribute is stored along with the L1 attributes to track the position of the L2 block within its epoch (starting at 0).

If you want you can clarify:
> Said otherwise, the `seq_number` is the the L2 block distance to the L2 block height that the L1 attributes were first introduced.